assembl/tests/views_tests/test_api2.py

```diff
@@ -899,3 +900,54 @@ def test_add_timeline_event(test_app, discussion):
     # check that the link was made in both directions
     assert phase1_data['next_event'] == phase2_data['@id']
     assert phase1_data['@type'] == 'DiscussionPhase'
+
+
+def test_phase1_export(proposals_with_sentiments, discussion, test_app):
+
+    THEMATIC_NAME = 0
+    QUESTION_ID = 1
+    QUESTION_TITLE = 2
+    POST_BODY = 3
+    POST_LIKE_COUNT = 4
+    POST_DISAGREE_COUNT = 5
+    POST_CREATOR_NAME = 6
+    POST_CREATOR_EMAIL = 7
+    POST_CREATION_DATE = 8
+    SENTIMENT_ACTOR_NAME = 9
+    SENTIMENT_ACTOR_EMAIL = 10
+    SENTIMENT_CREATION_DATE = 11
+
+    response = test_app.get(
+        '/data/Discussion/{}/phase1_csv_export'.format(discussion.id))
```

Comment:
Also, is there a template csv in the repo that this test is calling on? Or have you mocked it elsewhere?